user-interface/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectSummaryComponent.java

```diff
@@ -1,253 +1,537 @@
+    analyteBox.addClassName("fixed-medium-width");
+
+    var sectionContent = new SectionContent();
+    sectionContent.add(speciesBox);
+    sectionContent.add(specimenBox);
+    sectionContent.add(analyteBox);
+    sectionContent.addClassNames("horizontal-list", "gap-medium", "wrapping-flex-container");
+    experimentInformationSection.setContent(sectionContent);
   }
 
-  private Button editButton() {
-    Button editButton = new Button("Edit");
-    editButton.addClickListener(event -> openProjectInformationDialog());
-    return editButton;
+  private Div buildSpeciesInfo(List<Experiment> experiments) {
+    var ontologyTerms = extractSpecies(experiments);
+    return buildOntologyInfo(ontologyTerms);
   }
 
-  private Button exportButton() {
-    Button exortButton = new Button("Export as RO-Crate");
-    exortButton.addClickListener(event -> {
+  private Div buildSpecimenInfo(List<Experiment> experiments) {
+    var ontologyTerms = extractSpecimen(experiments);
+    return buildOntologyInfo(ontologyTerms);
+  }
+
+  private Div buildAnalyteInfo(List<Experiment> experiments) {
+    var ontologyTerms = extractAnalyte(experiments);
+    return buildOntologyInfo(ontologyTerms);
+  }
+
+  private Div buildOntologyInfo(List<OntologyTerm> terms) {
+    var container = new Div();
+    terms.stream().map(this::convert).forEach(container::add);
+    container.addClassNames("vertical-list", "gap-small");
+    return container;
+  }
+
+  private OntologyTermDisplay convert(OntologyTerm ontologyTerm) {
+    return new OntologyTermDisplay(ontologyTerm.getLabel(), ontologyTerm.getOboId(),
+        ontologyTerm.getClassIri());
+  }
+
+  private List<OntologyTerm> extractSpecies(List<Experiment> experiments) {
+    return experiments.stream().flatMap(experiment -> experiment.getSpecies().stream()).toList();
+  }
+
+  private List<OntologyTerm> extractSpecimen(List<Experiment> experiments) {
+    return experiments.stream().flatMap(experiment -> experiment.getSpecimens().stream()).toList();
+  }
+
+  private List<OntologyTerm> extractAnalyte(List<Experiment> experiments) {
+    return experiments.stream().flatMap(experiment -> experiment.getAnalytes().stream()).toList();
+  }
+
+  private void buildDesignSection(ProjectOverview projectInformation, Project project) {
+    var editButton = new Button("Edit");
+    editButton.addClickListener(listener -> {
+      editProjectDesignDialog = buildAndWireEditProjectDesign(project);
+      editProjectDesignDialog.open();
+      editProjectDesignDialog.addUpdateEventListener(event -> {
+        updateProjectDesign(context.projectId().orElseThrow(), event.content().orElseThrow());
+        reloadInformation(context);
+        editProjectDesignDialog.close();
+        var toast = notificationFactory.toast("project.updated.success",
+            new String[]{project.getProjectCode().value()}, getLocale());
+        toast.setDuration(Duration.ofSeconds(5));
+        toast.open();
```

Comment:
Since the toasts should behave the same independent on which section is edited this could be moved into its own method which is called by all buildSection methods?

Reply:
TBH, i think the toasts should behave equal for the same type in the complete app. I agree with you, that it makes no sense to configure them on a custom base every time we create them.